client/views/applicationState.html

```diff
@@ -386,7 +386,7 @@ <h2 data-bind="text: environment.env"></h2>
                         </span>
                         <span data-bind="text: title, click: $parent.appStateModel.sort" class= "cursor-pointer"
                             title="Toggle ascending/descending sort based on this column."></span>
-                        <span data-bind="visible: title == 'Configuration Path'" class="cursor-pointer"
+                        <span data-bind="visible: title == 'Application ID'" class="cursor-pointer"
```

Comment:
This currently shows the cursor for ceres/greeks/headnode in staging, however there aren't any dependencies? Not sure what's causing it. 
![image](https://cloud.githubusercontent.com/assets/5367814/6359303/b2009344-bc37-11e4-9652-837e11962e8c.png)